src/generics/Starter.java

```diff
@@ -0,0 +1,39 @@
+package generics;
+
+
+import java.util.*;
+
+/**
+ * Created by Komatoz on 17.01.2018.
+ */
+public class Starter {
+
+
+    public static void main(String[] args) {
+        System.out.println("Finish task3");
+    }
+
+    public static <T> int countEven(List<? extends Number> list){
+        int result = 0;
+        for (Number num : list) {
+            if (Math.IEEEremainder(Double.valueOf(num.toString()), 2) == 0) result++;
+        }
+        return result;
+    }
+
+    public static <T> void swapListElement(List<T> list1, List<T> list2, int i){
+        list1.set(i, list2.get(i));
```

Comment:
4. Create utility method, which takes two lists and index and replaces element at given position in the first collection with corresponding element from the second one, without possibility to do vice versa.

It shouldn't be possible to write `list2.set(i, list1.get(i));`. Can you try it?